packages/frontend-shared/src/locales/en-US.json

```diff
@@ -543,7 +543,7 @@
     "after": "After",
     "heresWhy": "here's why:",
     "renameAuto": {
-      "title": "We recommend automatically renaming your specs in this step",
+      "title": "We recommend automatically updating your specs in this step",
```

Comment:
"updating" sounds to broad to me, what we _recommend_ is definitely renaming. This title probably doesn't need a change.

Reply:
Good point, reverted